tests/test_10_use_cases.py

```diff
@@ -0,0 +1,210 @@
+
+            - Create a L2VPN between Tenet01:50 and Tenet03:50
+            - Check if that L2VPN status is UP (some wait may be required here)
+            - Run a PING test between hosts to make sure we have connectivity
+            - Config the Link Tenet01 — Tenet03 to down (some wait may be required here)
+            - Query the L2VPN and check the status to validate it is down
+            - Config the Link Tenet01 — Tenet03 to UP (some wait may be required here)
+            - Query the L2VPN and check the status to validate it is back UP
+            - Run a PING test between hosts to make sure we have connectivity
+        '''
+        api_url = SDX_CONTROLLER + '/l2vpn/1.0'
+        payload = {
+            "name": "Test L2VPN creation",
+            "endpoints": [
+                {"port_id": "urn:sdx:port:tenet.ac.za:Tenet01:50","vlan": "100"},
+                {"port_id": "urn:sdx:port:tenet.ac.za:Tenet03:50","vlan": "100"}
+            ]
+        }
+        response = requests.post(api_url, json=payload)
+        assert response.status_code == 201, response.text
+
+        # wait until status changes for under provisioning to up
+        time.sleep(5)
+
+        data = requests.get(api_url).json()
+        key = next(iter(data))
+        assert data[key]["status"] == "up"
+
+        h6, h8 = self.net.net.get('h6', 'h8')
+        h6.cmd('ip link add link %s name vlan100 type vlan id 100' % (h6.intfNames()[0]))
+        h6.cmd('ip link set up vlan100')
+        h6.cmd('ip addr add 10.1.1.6/24 dev vlan100')
+        h8.cmd('ip link add link %s name vlan100 type vlan id 100' % (h8.intfNames()[0]))
+        h8.cmd('ip link set up vlan100')
+        h8.cmd('ip addr add 10.1.1.8/24 dev vlan100')
+
+        # test connectivity
+        assert ', 0% packet loss,' in h6.cmd('ping -c4 10.1.1.8')
+
+        # set one link to down
+        self.net.net.configLinkStatus('Tenet01', 'Tenet03', 'down')
+
+        time.sleep(15)
+
+        '''
+            HERE: The following code verifies that the link was set to down, 
+            and that the current_path is Tenet01-Tenet03, 
+            so the test fails because the L2VPN status is still up.
+        '''
+
+        ## Beginning of verification code
+        api_url_topology = SDX_CONTROLLER + '/topology'
+        response = requests.get(api_url_topology)
+        data = response.json()
+        links = {link["id"]: link for link in data["links"]}
+        link1 = "urn:sdx:link:tenet.ac.za:Tenet01/2_Tenet03/2"
+        assert links[link1]["status"] == "down", str(links[link1])
+        ## End of verification code
+
+        data = requests.get(api_url).json()
+        key = next(iter(data))
+        ## Beginning of verification code
+        path = [item['port_id'] for item in data[key]['current_path']]
+        assert path == ['urn:sdx:port:tenet.ac.za:Tenet01:50', 'urn:sdx:port:tenet.ac.za:Tenet03:50']
```

Comment:
@gretelliz at this point the L2VPN should be down, right? In this case, I don't think we should validate the current_path (ideally, the current path should be empty -- that is how we do for Kytos, ie. if we cannot setup a path we leave the L2VPN down and empty path), or at least not before the status.

My suggestion is to invert lines 106-107 with 109. That way, we can first check the status and then the current path. For the current_path, let's also check for its emptiness, okay?

Reply:
@italovalcy, thanks for your review. 
That's the point. Initially, the `current_path` check wasn't in the test. It's a piece of code I added after seeing that the L2VPN status remained `up` when it should have changed to `down`. 
Then I checked the `current_path`. I assumed it should be empty, but instead, the path remains the same. 
That's why I added the lines of code between -## Beginning/End of verification code-, to illustrate this point.